pkg_specs/bitcoin-@variant.sps

```diff
@@ -49,6 +49,14 @@ The full path to the directory which will contain timechain data (blocks and cha
 Important: you need around 10GB of free space!
 """
 
+[config."bitcoin.conf".ivars.debuglogfile]
+type = "path"
+file_type = "regular"
+create = { mode = 755, owner = "$service", group = "$service" }
+default = "/var/log/bitcoin-{variant}/debug.log"
+priority = "medium"
+summary = "File containing the bitcoind debug logs"
```

Comment:
Would be nice to mention the network, e.g. "File containing the bitcoind debug logs {variant}"